include/arizona.hrl

```diff
@@ -1,4 +1,3 @@
--define(ARIZONA_LIVEVIEW(Str), (begin
-    {ok, Tree} = arizona_live_view:parse_str(Str, Macros),
-    Tree
-end)).
+-define(ARIZONA_LIVEVIEW(Macros, Str), (
+    arizona_live_view:parse_str(Str, Macros)
```

Comment:
Opacity less broken (and one step in the direction of "macroless", if possible).

Reply:
I'm all in for "macroless".